### PR TITLE
feat: add httpie completion spec

### LINKED ISSUE #462: [httpie] Add Spec
### Sanity checks

- [X] My issue relates to a specific CLI completion spec (e.g. `git checkout` is missing options in `git` completion spec). If your issue is more general, please create your issue here: [withfig/fig](https://github.com/withfig/fig/issues/new?assignees=&labels=NEED_TO_LABEL&template=main_issue_template.yml)
- [X] I have searched [github.com/withfig/autocomplete/issues](https://github.com/withfig/autocomplete/issues?q=) and there are no duplicates of my issue

### What CLI tool does this relate to?

httpie

### Which statement makes the most sense?

This completion spec does not exist

### Issue Details

https://github.com/httpie/httpie

### PR DESCRIPTION
**Pull request by cipi1965**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

This PR adds autocompletion support for httpie (HTTP client).

**What is the current behavior? (You can also link to an open issue here)**

Missing support (closes #462)

**What is the new behavior (if this is a feature change)?**

**Additional info:**

### CODE REVIEW

**Review by fedeci — CHANGES_REQUESTED** [review]
Since they are basically the same spec, can we just create one of the two and then import the other changing the name property before exporting?
```ts
import https from "./http";
https.name = "https";
export default https;
```

**Comment by cipi1965** [comment]
@fedeci done, could you please recheck?

**Comment by fedeci** [comment]
It should be ok now!

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Didn't know this CLI before, seems really useful! I left some comments to resolve

File: dev/https.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import completionSpec from "./http";
+completionSpec.name = "https";
+export default completionSpec;
```
[the remark this answers is not in the document]
Comment (fedeci) [reply dev/https.ts:3]: 🤔?

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks a lot!

**Comment by FAvE01** [comment]
#589

**Pull request merged**